plugin/pkg/scheduler/core/equivalence_cache.go

```diff
@@ -18,29 +18,34 @@ package core
 
 import (
 	"hash/fnv"
-
-	"github.com/golang/groupcache/lru"
-
 	"sync"
 
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/api/v1"
 	hashutil "k8s.io/kubernetes/pkg/util/hash"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/algorithm"
+
+	"github.com/golang/glog"
+	"github.com/golang/groupcache/lru"
 )
 
-// TODO(harryz) figure out the right number for this, 4096 may be too big
-const maxCacheEntries = 4096
+// we use predicate names as cache's key, its count is limited
+const maxCacheEntries = 100
```

Comment:
The ecache data structure in this patch is like:
```
eCache.Get(“podaffinity”).get(“EQUIVALENCE_HASH”) = {fit, reasons, invalid}
```
So the `maxCacheEntries` which describe entries of ecache should be less or equal to the total predicates counts of scheduler, it's a relatively fixed and small number. That's also why I did not use a knob.

I have also benchmarked different item like `EQUIVALENCE_HASH` as ecache's key, but it did not show any performance improvement while leaves `maxCacheEntries` undetermined.